🥇 Add Discord Server to README (#14)
SkriptLang/skript-worldguard: README.md

## Requirements
skript-worldguard is designed for Skript 2.6.3 and above.
It works with all Minecraft versions supported by Skript 2.6.3.

## Where to Download?

> For now, you can find and download alpha and beta releases at our [Skript Testing Discord server](https://discord.gg/ZPsZAg6ygu)

Reviewer: ```suggestion
For now, you can find and download alpha and beta releases on our [Skript Testing Discord](https://discord.gg/ZPsZAg6ygu).
```
